examples/shell/README.md

```diff
@@ -34,9 +34,9 @@ Done
 -   [exit](#exit)
 -   [help](#help)
 -   [otcli](README_OTCLI.md)
--   [ping](#ping)
+-   ping
```

Comment:
Remove the entire line? The command has been removed AFAIK.

Reply:
Fixed